doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -331,53 +451,62 @@ Catkin requires author information for publishing purposes
+  files to be generated tab and you can click on each of them for a
+  description of what they do.
 
-.. image:: setup_assistant_panda_done.png
+.. image:: setup_assistant_done.png
    :width: 700px
 
-* Congratulations!! - You are now done generating the configuration
-  files you need for MoveIt
+Congratulations! You are now done generating the configuration files you need for MoveIt.
 
+Build the panda_moveit_config package and run the demo
+------------------------------------------------------
+To build only the generated panda_moveit_config package and run the demo, follow these steps. ::
 
-What's Next
----------------
+   cd ~/ws_moveit2
+   colcon build --packages-select panda_moveit_config
+   source install/setup.bash
 
+Start the MoveIt demo to interactively plan and execute motions for the robot in RViz. :: 
 
-The MoveIt RViz plugin
+   ros2 launch panda_moveit_config demo.launch.py
 
-* Start looking at how you can use the generated configuration files
-  to play with MoveIt using the
-  :doc:`MoveIt RViz Plugin </doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial>`.
+Check out this `brief YouTube video <https://youtu.be/CXeF5KRa55Y>`_ for an example of how to command the robot to move to the pre-defined ready pose and execute open and close motions on the hand.
 
-Setup IKFast Inverse Kinematics Solver
+What's Next
+-----------
 
-* A faster IK solver than the default KDL solver, but takes some
-  additional steps to setup:
-  :doc:`Kinematics/IKFast </doc/examples/ikfast/ikfast_tutorial>`
+Get Started with MoveIt Motion Planning using RViz
 
-Additional Reading
----------------------
+* Learn how to use the generated configuration files to plan and visualize motion with MoveIt in RViz. 
+  Check out the `MoveIt Quickstart in Rviz tutorial <https://moveit.picknik.ai/humble/doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial.html>`_ for a step-by-step guide.
+
+Write Your First C++ MoveIt Application
 
-* See the :doc:`URDF and SRDF </doc/examples/urdf_srdf/urdf_srdf_tutorial>` page for more
+* Write your first C++ application using MoveIt with this `tutorial <https://moveit.picknik.ai/humble/doc/tutorials/your_first_project/your_first_project.html>`_, and familiarize yourself with the MoveGroupInterface and use it to plan, execute, and visualize motion plans for your robot from this `example <https://moveit.picknik.ai/humble/doc/examples/move_group_interface/move_group_interface_tutorial.html>`_. 
+
+URDF vs SRDF: Understand the Differences
+
+* See the `URDF and SRDF <https://moveit.picknik.ai/humble/doc/examples/urdf_srdf/urdf_srdf_tutorial.html>`_ page for more
   details on the components of the URDF and SRDF mentioned in this tutorial.
+
+Setup IKFast Inverse Kinematics Solver
+
+* A faster IK solver than the default KDL solver, but takes some
+  additional steps to setup `Kinematics/IKFast <https://moveit.picknik.ai/humble/doc/examples/ikfast/ikfast_tutorial.html>`_
```

Comment:
This link doesn't quite fit in the full sentence. Consider rewording or adding a : in between

Reply:
Thank you for the feedback. I have rephrased this sentence, but I would appreciate your opinion on whether it is suitable to keep. I took this point from the old documentation.

Reply:
I updated this sentence again and added a link to TRAC-IK.